### ISSUE TITLE
Event detection

### ISSUE THREAD

**Issue by zhikai-liu** [magland/mountainlab#188]
Hello, I am recently trying the ms3 branch for my data. I want to first compare the results with the old algorithm we use in the past, especially because we use a different algorithm for event detection (more complicated than threshold crossing). I already got the ms3 to work on some of my recordings. For the output file firings.mda, how can I get the fourth row of peak amplitudes (I saw in the documentation that it is optional)? I believe it would be helpful to get that info too!
Also, by the way, do you think I should build a new pipeline if I decide to modify the code and use a custom event detection algorithm?
Thanks,
Zhikai

**Comment by magland** [comment]
To create the amplitudes you need to use a processor. I think it's called "compute_amplitudes" but you may also look into "mountainsort.compute_amplitudes". One or both of those will produce a file with amplitudes int the 4th row of the firings file.

Regarding building a new pipeline. I think yes, you should. Soon we will release a new version that will provide a lot more flexibility in how to construct such pipelines and mix and match processors.

**Comment by zhikai-liu** [comment]
Thanks for your response!
I tried ms3 first
`: ./MountainLab/mountainlab/packages/mountainsort3/bin/mountainsort3.mp mountainsort.compute_amplitudes`
But it says 

> "Unexpected processor name: mountainsort.compute_amplitudes"

So I went to the older package 
`: ./packages/mountainsort2/bin/mountainsort2.mp mountainsort.compute_amplitudes`
There I can find the processor:

> Warning: Unable to open mda file for reading: 

> Warning: Unable to open mda file for writing:

I also checked `mountainsort3.mp.spec`, and it seems there is no `mountainsort.compute_amplitudes` in it. It can only be found in `mountainsort2.mp.spec`

So is it that the the new ms3 hasn't add the "compute_amplitudes" processor yet?
